### DISCUSSION TITLE
Error ‘ too many packets lost’ when crazyflie 2 near base station v2

### DISCUSSION THREAD

**Discussion by Shibaandcorgii** [bitcraze/discussions#52] [Q&A]
Hi! I have some issue about crazyflie + lighthouse deck and base station. I’ve been stucking with this for a while.
At first, this problem can be solved because we didn’t update the base station firmware. After we do some flying drone tests, the same problem occurred even using channel 80 or another channel. At first, I think because the base station where we set up is too near to the drone so that we decided to set up again but it still has the same problem.

**Comment by knmcguire** [comment]
Hi! We are actually on  a long weekend holiday now so won't be able to test this out... however, I did find a similar symptoms on a recent forum post: [Too many packets lost & lighthouse firmware](https://forum.bitcraze.io/viewtopic.php?t=5381). 

I hope to merge these two discussions together here since the forum will not be used anymore and next week we will look into find what the cause is!

**Comment by Shibaandcorgii** [comment]
We can solve  error ‘ too many packets lost ‘ by changed PA dongle . But when the crazyflie is taking off, the led lighthouse deck is orange and crash. I don’t know what’s wrong with it.

**Reply by knmcguire** [reply]
Alright. Let's discuss further for that in this discussion thread: https://github.com/orgs/bitcraze/discussions/71. For this discussion can mark your solution as the answer
